openmetadata-spec/src/main/resources/json/schema/entity/services/connections/metadata/openMetadataConnection.json

```diff
@@ -18,6 +18,24 @@
       "additionalProperties": {
         "type": "string"
       }
+    },
+    "authProvider": {
```

Comment:
we should also use this reference in ssoServiceType in ssoAuth.json , because that list is same as this with only difference of "openmetadata"

Reply:
@mohityadav766 our goal is to remove all the authentication providers for OM 
UI can ssoType for enabling authentication to SSO. But OM API auth will only support the JWT token auth. So we can keep these separate and remove all the references to other providers except for "openmetadata" ideally we should rename to JWTToken.